src/analysis/matrix.lean

```diff
@@ -132,6 +132,7 @@ end
 congr_arg coe $ nnnorm_diagonal v
 
 /-- Note this is safe as an instance as it carries no data. -/
+@[nolint fails_quickly]
 instance [nonempty n] [decidable_eq n] [has_one α] [norm_one_class α] :
   norm_one_class (matrix n n α) :=
 ⟨(norm_diagonal _).trans $ norm_one⟩
```

Comment:
My computer and Gitpod are both not powerful enough for me to diagnose the problem with `import all`. I could not figure it out by hand either.